docs/source/citation.rst

```diff
@@ -0,0 +1,123 @@
+.. _citation:
+
+======
+Citing
+======
+The pyiron integrated development environment (IDE) for computational materials science - pyiron IDE - is based on a flexible plugin infrastructure. So depending on which modules are used please cite the corresponding papers.
+
+*****************************
+pyiron paper (accepted)
```

Comment:
Here, we could switch to 'published' or rather just leave it out.

Reply:
Citing links to the original repo, so I'll throw the whole file.